src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -25,16 +25,18 @@
                 <shared:commentsPanel commentsForStudentsTables="${data.commentsForStudentTable}" courseId="${data.courseId}" forRecordsPage="${true}" />
                 <br>
                 <c:forEach items="${data.sessionNames}" var="fsName" varStatus="fbIndex">
-                    <div class="well well-plain student_feedback"
+                    <div class="student_feedback panel panel-default"
```

Comment:
I see, my apologies, I had overlooked that. Just leave it as it is for now then. I will create an issue regarding this.